lib/rules/selector-max-specificity/README.md

```diff
@@ -12,7 +12,7 @@ Visit the [Specificity Calculator](https://specificity.keegan.st) for visual rep
 
 This rule ignores selectors with variable interpolation (`#{$var}`, `@{var}`, `$(var)`).
 
-This rule ignores selectors containing the `:not()` or `:matches()` pseudo-classes.
+This rule handles selectors containing the `:not()` or `:matches()` pseudo-classes per [spec](https://drafts.csswg.org/selectors/#specificity-rules).
```

Comment:
I think we can drop this line completely as the user can assume it supports level 4 selectors. These sentences are generally used to inform the user of _limitations_ of the rule, when dealing with specs.